collection/lucas.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "lucas",
+    "meaning": "light",
+    "aliases": ["lu"],
+    "translations": {
```

Comment:
Hi @lucasp90,
Please make sure all the translation stars with a lower case letter.
Will gladly merge after needed changes. thanks!